packages/xrpl/test/models/escrowFinish.test.ts

```diff
@@ -20,6 +20,7 @@ describe('EscrowFinish', function () {
       Condition:
         'A0258020E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855810100',
       Fulfillment: 'A0028000',
+      CredentialIDs: [''],
```

Comment:
Can you specify a valid 256 byte hexadecimal string in the `CredentialIDs` list? I understand that this is not needed for unit tests, but this could serve as useful documentation for developers. Providing a valid example serves that purpose.